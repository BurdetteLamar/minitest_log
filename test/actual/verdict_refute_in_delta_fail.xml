<log>
  <summary_ verdicts='2' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_in_delta?' type='fail'>
    <section_ name='Args=[0, 0, 1]'>
      <verdict_ method='verdict_refute_in_delta?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Integer' value='0'/>
        <actual_ class='Integer' value='0'/>
        <delta_ class='Integer' value='1'/>
        <exception_ class='Minitest::Assertion' message='Expected |0 - 0| (0) to not be &lt;= 1.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[1, 1, 1]'>
      <verdict_ method='verdict_refute_in_delta?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Integer' value='1'/>
        <actual_ class='Integer' value='1'/>
        <delta_ class='Integer' value='1'/>
        <exception_ class='Minitest::Assertion' message='Expected |1 - 1| (0) to not be &lt;= 1.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
</log>
