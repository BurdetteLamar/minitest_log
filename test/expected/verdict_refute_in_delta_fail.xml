<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_refute_in_delta?' type='fail'>
    <section name='Args=[0, 0, 1]'>
      <verdict method='verdict_refute_in_delta?' outcome='failed' id='0' message='Message 0'>
        <exp_value>
          0
        </exp_value>
        <act_value>
          0
        </act_value>
        <delta>
          1
        </delta>
        <exception class='Minitest::Assertion'>
          <message>
            Expected |0 - 0| (0) to not be &lt;= 1.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[1, 1, 1]'>
      <verdict method='verdict_refute_in_delta?' outcome='failed' id='1' message='Message 1'>
        <exp_value>
          1
        </exp_value>
        <act_value>
          1
        </act_value>
        <delta>
          1
        </delta>
        <exception class='Minitest::Assertion'>
          <message>
            Expected |1 - 1| (0) to not be &lt;= 1.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
